jobrunner/lib/docker.py

```diff
@@ -158,7 +158,14 @@ def copy_to_volume(volume_name, source, dest, timeout=None):
     )
 
 
-def copy_from_volume(volume_name, source, dest):
+def copy_from_volume(volume_name, source, dest, timeout=None):
```

Comment:
Do you need to use the timeout somewhere?

Reply:
I do, George. I really do.